PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -1,3 +1,4 @@
 javaVersion=1.8
 sources=false
-intellij_version=IC-2017.1
\ No newline at end of file
+intellij_version=IC-2017.1
+dep_plugins=org.intellij.scala:2017.1.15
```

Comment:
Can we use maven dependency to resolve scala plugin build dependency?

Refer this: http://www.jetbrains.org/intellij/sdk/docs/plugin_repository/api/maven_interface.html